Brainduino/Brain.cpp

```diff
@@ -9,26 +9,26 @@
 
 Brain::Brain(Print *printer, Stream *stream_in, BrainDelegate *delegate)
 {
-    set(printer, stream_in, delegate, nullptr);
+    set(printer, stream_in, delegate, 0);
```

Comment:
That was my answer! :D https://github.com/brain-labs/brainduino/blob/2eb2810e949bbf9532b16a515cab1984ada783ea/Brainduino/Brain.h#L89